### PR TITLE
[MRG] all development done in a personal fork

### PR DESCRIPTION
**Pull request by untzag**
attempting to clarify my question for future developers

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ help and spread the knowledge.
 
 Contributions should follow these guidelines:
 
+* all development done in a personal fork;
```
Comment (betatim) [inline CONTRIBUTING.md]: -> all development is done in a personal fork ? Sounds a bit more like a real sentence :-/

**Comment by betatim** [comment]
Would it help to link to something like https://opensource.guide/how-to-contribute/#opening-a-pull-request?

**Comment by untzag** [comment]
@betatim I think your link is helpful---tried to incorporate it

**Comment by betatim** [comment]
Thanks :)

**Pull request merged**